src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -30,23 +30,75 @@
                                     </button>
                                 </form>
 
-                                @if (comment.Username == User.GetGitHubLogin())
-                                {
-                                    <div class="dropdown show">
-                                        <a class="btn btn-sm" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                                            ...
-                                        </a>
+                                <div class="dropdown show">
+                                    <a class="btn btn-sm" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                        ...
+                                    </a>
 
-                                        <div class="dropdown-menu">
+                                    <div class="dropdown-menu clickable">
+                                        @if (comment.Username == User.GetGitHubLogin())
+                                        {
                                             <form data-post-update="comments" asp-controller="Comments" asp-action="Delete" method="post" asp-route-reviewId="@Model.ReviewId">
-                                                <input type="hidden" name="commentId" value="@comment.CommentId"/>
+                                                <input type="hidden" name="commentId" value="@comment.CommentId" />
                                                 <button id="@comment.CommentId" type="submit" class="text-danger dropdown-item">Delete</button>
                                             </form>
                                             <a class="dropdown-item js-edit-comment">Edit</a>
+                                        }
+                                        <button type="button" class="btn dropdown-toggle dropdown-item" data-toggle="dropdown" aria-haspopup="true"
+                                                aria-expanded="false">
+                                            Create GitHub issue
+                                        </button>
+                                        <div class="dropdown-menu">
+                                            <table>
```

Comment:
Do we really need a table here?

Reply:
```
<div class="dropdown-item-text">
<a href="#" class="btn btn-link btn-sm js-github" data-repo="azure-sdk-for-net" data-repo-language="c#">.NET</a>
<a href="#" class="btn btn-link btn-sm js-github" data-repo="azure-sdk-for-python" data-repo-language="python">Python</a>
<a href="#" class="btn btn-link btn-sm js-github" data-repo="azure-sdk-for-java" data-repo-language="java">Java</a>
<a href="#" class="btn btn-link btn-sm js-github" data-repo="azure-sdk-for-js" data-repo-language="javascript">JS</a>
<a href="#" class="btn btn-link btn-sm js-github" data-repo="azure-sdk-for-c" data-repo-language="c">C</a>
<a href="#" class="btn btn-link btn-sm js-github" data-repo="azure-sdk-for-go" data-repo-language="go">Go</a>
</div>
```

Reply:
![image](https://user-images.githubusercontent.com/1697911/93904838-1df08600-fcaf-11ea-9d69-60c83b5df633.png)

Reply:
That looks weird IMO. I like having the table of two languages per row.